erigon-lib/txpool/fetch.go

```diff
@@ -177,7 +177,7 @@ func (f *Fetch) receiveMessage(ctx context.Context, sentryClient sentry.SentryCl
 	var req *sentry.InboundMessage
 	for req, err = stream.Recv(); ; req, err = stream.Recv() {
 		if err != nil {
-			f.logger.Error("[txpool.receiveMessage]", "err", err)
+			f.logger.Debug("[txpool.receiveMessage]", "err", err)
```

Comment:
also: I understand why need log error which we not return. But if we return - then it's job of caller to log it (otherwise we may log same thing multiple times).
if need add context to error: can use `fmt.Errorf("txpool.receiveMessage: %w", err)`

Reply:
return wrapped error instead of logging